sonic_api.yaml

```diff
@@ -2274,6 +2274,35 @@ definitions:
       ip_addr:
         type: string
         description: IP address of directly attached device to the switch
+  BfdEntry:
+    type: object
+    properties:
+      if_name:
+        type: string
+      tx_interval:
+        type: integer
+        description: interval that this system will use between transmission of control packets
```

Comment:
can this have default value? the same comments go to other parameters.

Reply:
Added.